src/Markup/Avalonia.Markup.Xaml/IAddChild.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Avalonia.Markup.Xaml
+{
+    public interface IAddChild
```

Comment:
That was probably a wrong place for the interface.

Reply:
Any suggestions?

Reply:
Well, `Avalonia.Markup.Xaml` references both `Avalonia.Controls` and `Avalonia.Base`. You probably want to implement `IAddChild` on `TextBlock` from `Avalonia.Controls`, so it should at least be in `Avalonia.Controls`.
Our `[Content]`, `[DataType]`, `[TemplateContent]` and several other attributes live in `Avalonia.Base`, so we could have `IAddChild` there as well